xonsh/environ.py

```diff
@@ -937,13 +937,13 @@ def _first_branch_timeout_message():
           file=sys.stderr)
 
 
-def current_branch(pad=True):
+def current_branch():
```

Comment:
Well, I know that people were taking partials of this function with pad set to false. I just don't what their rcs to break overnight. But if the people who where doing this say it is not a big deal, then we can go ahead an break it. I think having a warning for a couple of releases is a reasonable way to approach this. We don't claim stability (yet) but we also don't want to knowingly make peoples xonsh's fail to start.